server/BraveAlerterConfigurator.js

```diff
@@ -202,10 +231,14 @@ class BraveAlerterConfigurator {
     switch (fromAlertState) {
       case CHATBOT_STATE.STARTED:
       case CHATBOT_STATE.WAITING_FOR_REPLY:
-        returnMessage = t('incidentCategoryRequest', {
-          lng: language,
-          incidentCategories: this.createResponseStringFromIncidentCategories(incidentCategories, language),
-        })
+        if (toAlertState === CHATBOT_STATE.RESET) {
```

Comment:
These changes could be tested in `test/unit/BraveAlerterConfiguratorTest/getReturnMessageToRespondedByPhoneNumberTest.js`